book/plugins/general-abilities/url-schemes/url-schemes.md

```diff
@@ -0,0 +1,47 @@
+# URL Schemes
+
+The Zapp system allows that one or more URL schemes are defined for each version of the app. The URL scheme can be used to launch the app. Moreover, it can be used to launch different parts of the app such as specific screens or plugins. In the context of plugins, the Plugin Scheme Interface API can be implemented so as to launch the plugin by using a URL scheme. This API also allows custom parameters to be passed on to the plugin.
+
+## Typical Use
+
+Plugin Url schemes are useful for launching the plugin by using a link from a HTML page or a push notification.
```

Comment:
Also used in App builder to specify what page to open after a button click and for plugins to open other pages like after a hook or an onboarding page

Reply:
👍